src/System.Drawing.Common/tests/SystemFontsTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace System.Drawing.Tests
+{
+    public class SystemFontsTests
+    {
+        public static IEnumerable<object[]> SystemFonts_TestData()
+        {
+            yield return Font(() => SystemFonts.CaptionFont, "CaptionFont", "Segoe UI");
+            yield return Font(() => SystemFonts.IconTitleFont, "IconTitleFont", "Segoe UI");
+            yield return Font(() => SystemFonts.MenuFont, "MenuFont", "Segoe UI");
+            yield return Font(() => SystemFonts.MessageBoxFont, "MessageBoxFont", "Segoe UI");
+            yield return Font(() => SystemFonts.SmallCaptionFont, "SmallCaptionFont", "Segoe UI");
+            yield return Font(() => SystemFonts.StatusFont, "StatusFont", "Segoe UI");
+
+            bool isArabic = (GetSystemDefaultLCID() & 0x3ff) == 0x0001;
+            yield return Font(() => SystemFonts.DefaultFont, "DefaultFont", isArabic ? "Tahoma" : "Microsoft Sans Serif");
+
+            bool isJapanese = (GetSystemDefaultLCID() & 0x3ff) == 0x0011;
+            yield return Font(() => SystemFonts.DialogFont, "DialogFont", isJapanese ? "Microsoft Sans Serif" : "Tahoma");
+        }
+
+        public static object[] Font(Func<Font> fontThunk, string systemFontName, string windowsFontName) => new object[] { fontThunk, systemFontName, windowsFontName };
```

Comment:
Nit: Rather than `fontThunk`, I'd suggest `createFont` or `getFont` or something like that.